src/App/components/Card/index.js

```diff
@@ -11,20 +11,14 @@ const Card = ({ type, title, imgSrc, text, content, textSection, smallText, btnT
                 {title}
             </header>
```

Comment:
What was our conclusion on panel header class? Maybe this should follow the same solution?

Reply:
Remove only using <headers> and <footers>, use only classes. Sure i just forgot 😅